core-web/libs/edit-content/src/lib/components/dot-edit-content-aside/dot-edit-content-aside.component.html

```diff
@@ -0,0 +1,61 @@
+<div class="edit-content-aside--header">
+    <strong style="display: flex; align-items: center; gap: 0.3em">
+        <i class="pi pi-info-circle"></i>
+        Information</strong
+    >
+    <dot-copy-button
+        *ngIf="asideData.contentlet?.inode"
+        [copy]="asideData.contentlet?.inode"
+        [attr.data-testId]="'inode'"
+        [label]="asideData.contentlet?.inode.slice(0, 8)"></dot-copy-button>
+
+    <dot-api-link
+        *ngIf="asideData.contentlet?.stInode"
+        [href]="'api/v1/contenttype/id/' + asideData.contentlet?.stInode"></dot-api-link>
+</div>
+
+<div class="edit-content-aside--item">
+    <span>Content State</span>
+
+    <ng-container [ngSwitch]="true">
+        <p-chip
+            *ngSwitchCase="asideData.contentlet?.live && !asideData.contentlet?.working"
+            [styleClass]="'p-chip-success p-chip-sm'"
+            label="Published"></p-chip>
+        <p-chip
+            *ngSwitchCase="asideData.contentlet?.working && !asideData.contentlet?.live"
+            [styleClass]="'p-chip-sm'"
+            label="Draft"></p-chip>
+        <p-chip
+            *ngSwitchCase="asideData.contentlet?.archived"
+            [styleClass]="'p-chip-gray p-chip-sm'"
+            label="Archived"></p-chip>
+        <p-chip
+            *ngSwitchCase="asideData.contentlet?.working && asideData.contentlet?.live"
+            [styleClass]="'p-chip-pink p-chip-sm'"
+            label="Revision"></p-chip>
+    </ng-container>
```

Comment:
You have logic on the template, pass that to the store, and use the label (status) with a map in the component to define the classes (could be). Something like:
```
const enum DotEditContentStatus {
    DRAFT = 'draft',
    ARCHIVED = 'archived',
    REVISION = 'revision',
    PUBLISHED = 'published'
}
```
at your component
```
statusMap: Record<DotEditContentStatus, { label: string; clazz: string }> = {
        archived: {
            label: 'Archived', <- you can use the service to translate this directly here
            clazz: 'p-chip-gray p-chip-sm'
        }
    };
```
Your HTML
```
<p-chip
	            [styleClass]="statusMap[aStatusFromTheStore].claszz"
	            label="statusMap[aStatusFromTheStore].label" />
```
it is only an idea.

Reply:
Note that the styleClasses change as well, maybe we can calculate this in the `.ts` file and create an options object to avoid the usage of `ngSwitch` but still, I don't think it's worth effort unless we need to add more states to this or more options

Reply:
@zJaaal I think it's worth the effort. Using the state of a `contentlet` is something we want to show to the user. Currently, we use it in several places as `Content Search`/`Site Browser`.